conda/activate.py

```diff
@@ -87,6 +91,8 @@ def __init__(self, shell, arguments=None):
             self.export_var_tmpl = "$%s = '%s'"
             self.run_script_tmpl = 'source "%s"'
 
+            self.deactivatepost_tmpl = """!(@(_CONDA_EXE) shell.xonsh deactivatepost)"""
```

Comment:
Looking at https://github.com/conda/conda/blob/4.5.0rc0/conda/shell/conda.xsh#L24-L38, it seems we need some more code for `xonsh` to properly source the output.